smartsim/experiment.py

```diff
@@ -467,35 +467,35 @@ def summary(self):
         The summary will show each instance that has been
         launched and completed in this ``Experiment``
 
-        :return: pandas Dataframe of ``Experiment`` history
-        :rtype: pd.DataFrame
+        :return: tabulate string of ``Experiment`` history
+        :rtype: str
         """
-        index = 0
-        df = pd.DataFrame(
-            columns=[
-                "Name",
-                "Entity-Type",
-                "JobID",
-                "RunID",
-                "Time",
-                "Status",
-                "Returncode",
-            ]
-        )
+        values = []
+        headers = [
+            "Name",
+            "Entity-Type",
+            "JobID",
+            "RunID",
+            "Time",
+            "Status",
+            "Returncode",
+        ]
+
         # TODO should this include running jobs?
         for job in self._control._jobs.completed.values():
             for run in range(job.history.runs + 1):
-                df.loc[index] = [
-                    job.entity.name,
-                    job.entity.type,
-                    job.history.jids[run],
-                    run,
-                    job.history.job_times[run],
-                    job.history.statuses[run],
-                    job.history.returns[run],
-                ]
-                index += 1
-        return df
+                values.append(
+                    [
+                        job.entity.name,
+                        job.entity.type,
+                        job.history.jids[run],
+                        run,
+                        job.history.job_times[run],
+                        job.history.statuses[run],
+                        job.history.returns[run],
+                    ]
+                )
+        return tabulate(values, headers, showindex=True, tablefmt="plain")
```

Comment:
lets set the default to ``github``. so the added parameter will be a keyword arg like ``format=github`` and then that input parameter will be passed here.